cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationConfiguration.kt

```diff
@@ -372,6 +385,26 @@ private constructor(
             return this
         }
 
+        /** Produces the [Pass] with the given name. */
+        fun producePass(passName: String): Pass {
+            val list = passList.map { Pair(it.simpleName, it) }.toMap()
+            if (!list.contains(passName)) {
+                throw ConfigurationException("Asked to produce unknown pass")
+            }
+            return list[passName]!!.createInstance()
+        }
+
+        /** Returns the list of available passes that can be registered. */
+        fun getPassList(): List<String> {
```

Comment:
Should be converted to a (virtual) property instead of a getter function

Reply:
Resolved in 979866b61.